.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
are these workflows free of charge?

Reply:
For public repositories they are free:
https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions